.github/workflows/release.yml

```diff
@@ -63,6 +63,51 @@ jobs:
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
 
+  # Build and push the edge image on every master push
+  # Use docker to build and push the amd64 image directly.
+  # No need to use depot without arm64 support
```

Comment:
```suggestion
  # Build and push the edge image on every master push.
  # Use official docker workflows since we only need to build amd64 images.
```